### ISSUE TITLE
Telegram Login

### ISSUE THREAD

**Issue by AKTanara** [shamhi/HamsterKombatBot#2358]
I have hard time understanding the reason behind reason this bot needs Telegram Login!
I have created a code that only uses `Bearer+Access token` and doesn't need any telegram account login!
This bot is also acquires Bearer and uses that to communicate with server:
tapper.py:
**`http_client.headers['Authorization'] = f"Bearer {access_token}"`** 

Why it do not ask just for Bearer?

Thanks for sharing

**Comment by ivan52605** [comment]
сессия телеграмм нужна на случай обновления токена это же автоматизация верно? в других проектах (тот же блум) иногда просит обновление токена так что привязка к сессии телеграмма это даже хорошо

**Comment by shamhi** [comment]
> сессия телеграмм нужна на случай обновления токена это же автоматизация верно? в других проектах (тот же блум) иногда просит обновление токена так что привязка к сессии телеграмма это даже хорошо

Да, если всегда использовать один и тот же initData, то это палевно
Некоторые игры также проверяют &auth_date=1725692666 и если например с момента авторизации прошло 3 часа, то не пропускают
Также видел, что не пропускают за использование одной initData во второй раз

**Comment by AKTanara** [comment]
@ivan52605 @shamhi Thanks for the explanation. I think at least for Hamster Kombat this is not the case because authorizations tokens haven't changed since day 1. So I think it might be a good idea to have the option of providing Bearer Token instead of logging in. No?

**Comment by ivan52605** [comment]
Для хомяка да один токен на всю жизнь но можно просто выбрать тип авторизации либо привязка с сесии либо токен хомяка

**Comment by ivan52605** [comment]
хотя щас хомяк стад добавлять телеграмм id а конец токена

**Comment by AKTanara** [comment]
> For a hamster, yes, one token for life, but you can simply choose the authorization type, either binding from a session or a hamster token
I tried but bot returned error.
What is the format of the token should be given to the bot?
